backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/UserService.java

```diff
@@ -197,6 +205,19 @@ public boolean verifyEmail(String userId, String code){
 		return true;
 	}
 
+	public User createUser(String userEmail, List<UserAttributeModel> userAttributesModel){
+		//Verify Email
+		Assert.hasText(userEmail, "userEmail is required; it must not be null or blank");
+
+		Collection<UserAttributeEntity> userAttributesCollection = new ArrayList<UserAttributeEntity>();
+		
+		for(UserAttributeModel e : userAttributesModel){
+			userAttributesCollection.add(new UserAttributeEntityBuilder().name(e.getName()).value(e.getValue()).build());
+		}
+		return userMapper.toUser(userRepository.save(new UserEntityBuilder().email(userEmail)
+			.userAttributes(userAttributesCollection).build()));
```

Comment:
Use a mapper method to convert from `User` to `UserEntity`. Only in rare cases should you ever have to use `UserEntityBuilder`.